src/scene/vcFolder.cpp

```diff
@@ -391,3 +397,13 @@ void vcFolder::Cleanup(vcState *pProgramState)
     pNode = pNode->pNextSibling;
   }
 }
+
+vdkProjectNode *vcFolder::RemoveCurrentNode(vcState *pProgramState, vdkProjectNode *pNode)
+{
+  if (m_pNode->pFirstChild == pNode)
+    m_pNode->pFirstChild = pNode->pNextSibling;
```

Comment:
@OsiJr can you do this? If not, should these pointers be `const`?

Reply:
Deleted.